go/chat/convloader.go

```diff
@@ -448,15 +453,46 @@ func (b *BackgroundConvLoader) load(ictx context.Context, task clTask, uid grego
 		b.Debug(ctx, "load: invoking post load hook on job: %s", job)
 		job.PostLoadHook(ctx, tv, job)
 	}
-
-	// if testing, put the convID on the loads channel
-	if b.loads != nil {
-		b.Debug(ctx, "load: putting convID %s on loads chan", job.ConvID)
-		b.loads <- job.ConvID
+	// if testing, put the convID on the loadTestCh
+	if b.loadTestCh != nil {
+		b.Debug(ctx, "load: putting convID %s on loadTestCh", job.ConvID)
+		b.loadTestCh <- job.ConvID
 	}
+
 	return nil
 }
 
+// Called periodically by the fastChatChecks ticker in service/main
+func (b *BackgroundConvLoader) QueueEphemeralPurges(ctx context.Context) {
```

Comment:
I'd prefer for this to not be a function on the conv loader. Maybe in `utils` somewhere? Or just next to the loop that is driving it?